### ISSUE TITLE
Supporting the Node.js 2018 User Survey

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#8]
Somebody, I assume @hackygolucky, launched the 2018 User Survey today:

* [Announcement on Medium](https://medium.com/@nodejs/take-our-node-js-2018-user-survey-965011560edb)
* [Direct link to Survey](https://www.surveymonkey.com/r/NodejsUserSurvey)

Let's discuss how we can support this survey effort and bring attention to it in the [areas of focus](https://github.com/nodejs/user-feedback#current-members) we have identified (Enterprise Users, Meetups and Individual Members). A few topics for discussion:

1) Node.js 2018 User Survey objectives
2) 2018 timeline and completion deadline
3) 2016 and 2017 Survey objectives, participation data and results
4) What additional context can we convey to our constituencies of interest to encourage participation and gathering quality data?
5) How and where can we target outreach?

**Comment by dshaw** [comment]
2018 User Survey feedback: “Survey is too long. I’d like to help but after 15 minutes, I gave up.” - https://twitter.com/jaredowenbeck/status/935215157009174528

**Comment by hackygolucky** [comment]
The survey was launched during Node Interactive back in October by the Node.js Foundation team. To provide some context about what is already being worked on:  requested translations and asks for help with promoting in the ecosystem of communities on Twitter and Medium.  We've also asked the corporate members along with the Marketing Committee specifically, to help with promotion by providing links and some copy. 

We also talked a bit about this in the last @nodejs/community-committee meeting, where we recognized that the work happening in regards to translation communities could be taken even further and will be incredibly necessary to gauge our global audience needs. 

We seek out input from a number of groups in the foundation and ecosystem every year for what should be focused for information gathering, so it'd be great to receive that feedback from y'all. The Node.js User Survey is a project of the Node.js Foundation and driven ultimately by Greg Wallace with my assistance in adjusting for prior feedback, technical updates, and survey science changes to make sure we're collecting relevant data in a way that our users will actually engage in providing it.

The survey launches every year at Node Interactive.
